app/src/main/java/ru/suslovalex/androidacademyapp/FragmentMoviesList.kt

```diff
@@ -0,0 +1,45 @@
+package ru.suslovalex.androidacademyapp
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+
+class FragmentMoviesList: Fragment() {
+
+    private var itemListener: ClickItemListener? = null
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.fragment_movies_list, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        view.findViewById<View>(R.id.item).setOnClickListener {
+            itemListener?.moveToMovieDetails()
```

Comment:
@AleksandrAlex No need to add listener here, you can directly call `activity?.supportFragmentManager` here